src/BoundaryConditions/polar_boundary_condition.jl

```diff
@@ -7,17 +7,31 @@ end
 
 Adapt.adapt_structure(to, pv::PolarValue) = PolarValue(Adapt.adapt(to, pv.data), nothing)
 
-const PolarBoundaryCondition{V} = BoundaryCondition{<:Value, <:PolarValue}
+const PolarValueBoundaryCondition{V} = BoundaryCondition{<:Value, <:PolarValue}
+const PolarOpenBoundaryCondition{V}  = BoundaryCondition{<:Open,  <:PolarValue}
 
-function PolarBoundaryCondition(grid, side, zloc)
+function PolarValueBoundaryCondition(grid, side, LZ)
     FT   = eltype(grid)
-    loc  = (Nothing, Nothing, zloc)
+    loc  = (Nothing, Nothing, LZ)
     data = new_data(FT, grid, loc)
     return ValueBoundaryCondition(PolarValue(data, side))
 end
 
+function PolarOpenBoundaryCondition(grid, side, LZ)
+    FT   = eltype(grid)
+    loc  = (Nothing, Nothing, LZ)
+    data = new_data(FT, grid, loc)
+    return OpenBoundaryCondition(PolarValue(data, side))
+end
+
+const PolarBoundaryCondition = Union{PolarValueBoundaryCondition, PolarOpenBoundaryCondition}
+
+PolarBoundaryCondition(grid, size, ::Nothing, LZ) = nothing
```

Comment:
This is wrong --- we call a constructor `PolarBoundaryCondition`, but we get `nothing` in result. We shouldn't write this because it can lead to confusion.

To express this we need to use the terminology `polar_boundary_condition` or `maybe_polar_boundary_condition`.